core/tabs/applications-setup/developer-tools/zed.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh -e
+
+. ../../common-script.sh
+
+installZed() {
+    if ! command_exists dev.zed.Zed && ! command_exists zed && ! command_exists zeditor; then
```

Comment:
```suggestion
    if ! command_exists dev.zed.Zed zed zeditor; then
```

this can actually just be put on one line instead of using &&